transit/transit.py

```diff
@@ -365,7 +365,7 @@ def omega(self, hp=0.5*np.pi):
 
     @omega.setter
     def omega(self, v, hp=0.5*np.pi):
-        self.pomga = v - hp
+        self.pomega = v - hp
```

Comment:
Fixing this stupid typo is the key here. As soon as this is fixed, the problem will go away because this will be called when `omega` is set (even the first time).